bin/plotting/pycbc_plot_singles_vs_params

```diff
@@ -84,12 +84,32 @@ opts = parser.parse_args()
 
 logging.basicConfig(format='%(asctime)s %(message)s', level=logging.INFO)
 
-snr_filter = '(self.snr>%f)' % (opts.min_snr) if opts.min_snr > 0. else None 
-filts = [f for f in [snr_filter, opts.filter_string] if f is not None]
+data_mask = None
+if opts.min_snr > 0:
+    with pycbc.io.HFile(opts.single_trig_file, 'r') as trig_file:
+        n_triggers_orig = trig_file[f'{opts.detector}/snr'].size
+        logging.info("Trigger file has %d triggers", n_triggers_orig)
+        logging.info('Generating trigger mask (on SNR)')
+        idx, _ = trig_file.select(
+            lambda snr: snr >= opts.min_snr,
+            f'{opts.detector}/snr',
+            return_indices=True
+        )
+        data_mask = np.zeros(n_triggers_orig, dtype=bool)
+        data_mask[idx] = True
+
+filts = [f for f in [opts.filter_string] if f is not None]
```

Comment:
Is the square bracket surrounding `opts.filter_string` intended here? This line does not make much sense to me (but I am quite tired right now…)

Reply:
This appears the same as the existing code and `join` works with a (non-empty) list.

Reply:
(could be a tuple as well, no difference to the outcome)

Reply:
The code does work, but I think having moved the SNR mask, the following two lines:
```
filts = [f for f in [opts.filter_string] if f is not None]
filter_func = ' & '.join(filts) if filts else None
```
basically reduce to:
```
filter_func = opts.filter_string
```
I'll just remove these lines and send opts.filter_string directly to the relevant function call. Then I'll merge unless there are objections.

Reply:
ah, I missed the point that this previously added the SNR cut to the filter option and now doesn't ..